ppr-ui/src/utils/format-helper.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Helper function to convert string to HTML format
+ * @param stringToFormat
+ */
+export function toHtml (stringToFormat: string) {
+  return stringToFormat.replace(/\r?\n/g, '<br />') // convert new line chars to html break lines
```

Comment:
is `<br />` the same as `<br>` 
Im not sure it needs to be self closing

Reply:
Updated as not self closing